docs/RFCS/range_repair.md

```diff
@@ -0,0 +1,94 @@
+- Feature Name: Range Repair
```

Comment:
Yeah, it's just pruning dead replicas and suggesting adding new ones.  I could call it the `Replica Pruning Service`.